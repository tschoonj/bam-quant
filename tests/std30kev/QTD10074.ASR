File: C:\xrfdaten\2014KW30\chris\std30kev\std10074.spe
Start: 22.07.2014 10:24:29
Stop: 22.07.2014 10:26:10
BESSY: 298.9671147 297.9550093
IONI: 1.951292E-11 1.94369E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97904
Run: 100000
Dead: 2
ICR: 710
OCR: 0
Accumulated BESSY: 59692.0806305
Accumulated Ioni: 3.89341323E-09
N Accumulation: 200
Normfactor:  5.2468550e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: W
exp_x 72.7203979492188 72.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  74   2      8.392      1799.        45.      30.75
