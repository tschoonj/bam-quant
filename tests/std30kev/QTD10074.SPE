$DATA:
0 2000
23054
14629
6283
1854
373
59
5
1
0
0
0
0
1
0
0
0
10
26
23
28
23
21
20
21
14
15
15
23
19
31
23
7
17
15
13
20
12
23
15
12
22
21
13
16
19
13
17
9
11
13
17
18
15
11
16
16
13
10
10
16
14
20
11
14
14
15
11
13
15
10
8
15
13
9
15
12
13
13
9
11
7
11
12
14
7
6
8
9
8
8
6
6
16
14
11
5
8
7
12
11
18
7
20
5
16
10
15
9
7
8
12
5
12
8
4
14
9
11
2
13
11
8
7
7
8
7
9
7
7
11
13
8
7
14
11
7
16
10
11
14
6
5
4
16
8
11
7
3
4
6
9
12
10
10
11
9
8
14
10
11
10
14
8
8
9
9
7
11
7
9
4
8
4
8
6
5
9
10
9
11
6
11
6
9
7
7
8
10
10
8
6
5
13
5
5
12
13
7
6
8
6
6
6
8
7
11
11
6
10
6
3
6
7
5
6
2
10
6
6
9
12
3
7
4
11
5
7
3
7
6
7
2
3
7
6
6
3
7
7
3
4
5
5
3
7
6
8
3
4
3
5
0
3
2
4
2
4
3
1
6
6
7
1
5
7
9
3
9
6
2
2
7
5
7
6
5
4
5
1
4
5
4
1
2
7
2
4
10
4
6
5
7
2
7
6
2
7
3
6
6
5
7
4
3
1
3
6
6
7
4
4
5
5
4
9
9
10
6
5
9
4
5
3
5
3
6
2
4
3
6
6
3
6
6
5
3
5
7
4
2
5
5
3
2
4
4
9
6
7
4
4
7
5
1
5
5
7
6
7
4
3
2
4
2
6
9
5
7
6
7
2
4
2
7
5
6
7
3
6
8
10
0
7
2
9
5
4
6
7
11
8
9
9
7
7
4
7
6
6
12
5
3
7
11
10
10
16
22
34
45
79
124
156
223
243
263
282
236
192
158
123
97
58
40
26
19
7
15
19
10
9
18
17
14
10
20
16
14
20
9
6
7
10
8
7
10
6
13
12
9
13
8
19
14
19
14
18
18
10
23
29
23
30
50
49
73
110
175
231
354
408
505
655
764
1022
1328
1481
1741
1986
1980
1866
1737
1545
1426
1165
1100
1050
931
914
902
894
889
859
862
789
719
626
502
369
293
200
134
97
94
66
75
65
70
45
55
44
33
28
24
16
19
10
8
10
12
10
11
10
8
5
6
7
10
7
7
7
11
8
10
17
6
22
26
25
33
35
26
41
42
47
37
43
42
63
56
74
125
201
271
391
445
658
688
790
838
745
695
565
513
360
265
210
180
167
167
193
269
325
378
431
463
486
477
454
413
357
282
220
172
118
82
68
50
40
37
56
66
65
70
94
99
115
115
119
108
80
66
52
29
26
24
13
7
12
10
10
7
6
8
3
5
3
7
7
2
6
4
4
4
7
8
6
4
11
9
11
7
10
1
7
4
11
7
12
11
4
4
8
8
8
6
4
3
5
2
3
14
12
6
7
7
3
13
2
9
9
4
5
6
9
5
8
7
8
4
11
8
12
10
8
12
10
9
11
1
5
11
9
2
4
6
10
4
8
9
5
11
13
10
14
6
15
11
8
10
11
3
11
10
16
12
9
7
8
12
3
3
9
12
8
8
15
7
10
6
6
12
6
8
5
4
14
7
13
12
9
7
11
9
14
10
6
8
14
7
11
14
14
6
8
13
18
12
11
13
7
10
7
16
6
13
11
18
4
7
7
16
12
17
11
11
14
16
10
19
22
27
31
35
38
41
63
86
92
80
77
95
87
70
54
49
42
30
21
23
12
12
14
8
13
13
13
8
10
6
12
13
14
14
14
14
6
9
13
9
14
14
16
9
18
14
8
13
15
11
13
15
5
12
13
7
12
9
10
13
14
8
8
11
9
11
14
9
14
5
8
6
8
14
7
10
10
8
15
14
16
15
11
8
14
8
22
15
14
13
16
19
12
27
13
21
18
14
24
24
27
16
18
13
27
21
23
13
15
21
11
10
16
12
14
12
9
12
13
16
12
13
8
8
14
14
11
17
13
11
5
10
19
9
13
15
3
11
13
5
14
4
11
4
8
11
7
14
8
10
13
14
3
7
9
8
13
11
8
11
8
6
16
13
12
13
15
10
10
13
11
12
12
9
9
15
12
15
9
14
12
11
8
9
3
11
9
11
11
13
9
5
11
10
13
14
6
12
10
10
12
16
13
11
12
9
4
6
15
6
11
12
18
9
8
8
4
13
9
8
9
15
8
10
3
15
8
12
10
14
12
11
13
11
17
8
9
11
8
10
7
10
6
9
9
9
7
4
8
6
7
8
6
4
9
8
10
10
10
9
7
6
13
10
7
7
7
10
5
9
6
15
8
5
9
11
4
6
10
10
8
10
5
4
9
8
11
3
6
8
11
10
10
10
4
7
6
7
5
14
6
11
6
10
5
7
6
9
6
4
11
12
10
7
12
5
5
4
3
6
7
2
5
7
5
10
4
5
4
10
6
8
8
10
13
15
4
10
5
13
9
7
6
1
7
8
5
3
4
5
4
9
6
11
7
8
7
12
4
3
6
10
5
3
4
4
7
6
6
5
3
3
6
8
8
7
4
4
7
7
8
5
6
10
5
8
1
6
8
5
3
3
5
3
8
8
6
4
7
3
3
6
5
5
4
8
2
11
5
4
6
4
4
5
5
3
3
5
7
8
4
7
1
4
8
7
2
2
3
4
3
4
5
4
10
4
3
5
7
1
7
1
3
5
1
2
4
6
6
4
5
1
5
4
4
2
5
5
5
5
5
3
7
5
11
7
8
4
11
8
6
7
10
6
7
7
9
11
10
9
15
12
10
11
12
9
5
6
6
9
3
5
5
3
3
2
2
3
4
3
1
2
2
3
1
2
2
3
1
2
0
2
0
3
1
3
5
0
1
1
2
2
1
6
1
2
4
1
1
0
5
6
1
3
4
3
1
3
5
2
2
2
2
2
0
0
3
9
0
0
3
0
1
1
3
1
2
3
1
3
1
4
2
6
3
5
1
3
1
6
5
5
6
3
3
4
3
4
1
3
4
1
0
3
5
4
3
3
2
5
4
6
4
4
1
2
6
1
3
3
1
5
1
2
2
4
3
2
3
4
3
6
3
3
3
2
2
2
4
3
5
2
3
0
2
2
3
3
0
0
3
5
6
4
0
2
4
6
7
3
4
5
4
5
2
2
1
2
2
2
5
1
3
2
1
2
3
2
1
4
6
2
3
4
2
2
4
6
3
6
3
3
5
2
4
1
6
3
2
2
2
4
4
3
5
1
4
3
2
3
2
7
3
3
6
2
4
1
6
4
4
4
3
2
5
4
3
3
5
4
4
6
6
6
1
3
4
3
1
7
4
4
6
4
3
2
6
6
5
1
5
10
4
6
5
5
4
5
4
2
3
12
5
2
5
5
8
5
11
3
7
6
5
6
4
9
4
8
5
3
2
1
3
7
4
6
4
4
4
6
8
5
11
9
8
8
8
11
10
6
10
4
8
8
6
8
4
9
4
7
11
7
6
4
9
9
7
13
9
6
10
1
10
10
4
7
7
9
6
10
7
7
6
5
9
9
9
10
5
8
7
16
10
10
9
9
9
12
12
9
10
10
9
10
20
11
7
14
9
12
6
13
7
11
10
10
10
16
16
9
17
18
13
4
9
18
15
11
15
11
12
9
16
14
10
17
13
13
11
8
15
19
14
8
10
5
12
11
10
12
12
15
11
16
18
13
12
11
11
11
9
9
11
5
14
5
13
11
16
14
11
16
16
11
14
11
12
10
21
18
17
19
7
20
7
15
18
19
12
13
16
18
24
21
17
11
14
19
9
16
19
15
21
15
22
12
21
14
22
24
26
17
22
26
30
27
30
48
30
38
56
53
61
69
51
69
93
77
88
91
98
96
84
107
118
103
107
105
116
120
92
131
128
121
106
109
114
94
101
83
89
83
72
54
78
69
48
58
57
42
39
34
18
19
14
26
6
8
8
13
5
3
4
0
4
4
3
1
1
0
2
1
1
0
1
0
0
0
0
1
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10074.spe
Start: 22.07.2014 10:24:29
Stop: 22.07.2014 10:26:10
BESSY: 298.9671147 297.9550093
IONI: 1.951292E-11 1.94369E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97904
Run: 100000
Dead: 2
ICR: 710
OCR: 0
Accumulated BESSY: 59692.0806305
Accumulated Ioni: 3.89341323E-09
N Accumulation: 200
Normfactor:  5.2468550e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: W
 
exp_x 72.7203979492188 72.7204
exp_y 65.4657974243164 65.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  74   2      8.392      1799.        45.      30.75
