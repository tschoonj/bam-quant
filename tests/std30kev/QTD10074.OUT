     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:58  บ   
     บ  Spectrum: QTD10074.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        5 iterations done    บ   
     บ  ChiSquare =    10.8   last change =   .07%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ     -127.2 ๑   100.0   บ    -127.2 ๑     8.2   บ   
     บ  GAIN (eV/ch)  บ     20.360 ๑   2.000   บ    20.360 ๑    .017   บ   
     บ det NOISE (eV) บ      132.1 ๑    40.0   บ     132.1 ๑    24.4   บ   
     บ  FANO factor   บ       .119 ๑    .050   บ      .119 ๑    .022   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 W -L1                             19554. ๑    214.           บ   
     บ    L1M3        9.819    .41627        8140. ๑     90.    112.5  บ   
     บ              488.523    206.09         127.           9.76E-01  บ   
     บ    L1M2        9.525    .32896        6433. ๑     71.     28.5  บ   
     บ              474.083    204.27         126.           9.74E-01  บ   
     บ    L1N3       11.674    .11300        2210. ๑     27.     26.5  บ   
     บ              579.634    217.26         148.           9.85E-01  บ   
     บ    L1N2       11.608    .08415        1646. ๑     22.     31.0  บ   
     บ              576.393    216.87         148.           9.85E-01  บ   
     บ    L1O3       12.059    .03006         588. ๑     14.     12.3  บ   
     บ              598.544    219.50         149.           9.83E-01  บ   
     บ    L1M5       10.291    .01442         282. ๑     12.     18.9  บ   
     บ              511.706    208.99         129.           9.79E-01  บ   
     บ    L1M4       10.228    .00961         188. ๑     12.     52.8  บ   
     บ              508.612    208.61         129.           9.78E-01  บ   
     บ    L1N5       11.851    .00353          69. ๑     12.     13.9  บ   
     บ              588.328    218.29         148.           9.86E-01  บ   
     บ                                                                 บ   
     บ  2 W -L2                             19824. ๑    215.           บ   
     บ    L2M4        9.672    .80449       15949. ๑    173.     51.9  บ   
     บ              481.303    205.18         127.           9.75E-01  บ   
     บ    L2N4       11.286    .16044        3181. ๑     36.    179.1  บ   
     บ              560.577    214.97         132.           9.84E-01  บ   
     บ    L2M1        8.724    .02168         430. ๑     12.     52.3  บ   
     บ              434.740    199.21         121.           9.66E-01  บ   
     บ    L2N1       10.949    .00553         110. ๑     12.     14.5  บ   
     บ              544.025    212.96         131.           9.82E-01  บ   
     บ    L2O4       11.539    .00615         122. ๑     12.     53.6  บ   
     บ              573.004    216.46         148.           9.85E-01  บ   
     บ    L2O1       11.468    .00101          20. ๑     12.    117.3  บ   
     บ              569.516    216.05         147.           9.84E-01  บ   
     บ    L2M3        9.261    .00070          14. ๑     11.     10.3  บ   
     บ              461.116    202.62         124.           9.71E-01  บ   
     บ                                                                 บ   
     บ  3 W -L3                              2263. ๑     57.           บ   
     บ    L3M5        8.398    .71392        1616. ๑     42.     30.7  บ   
     บ              418.728    197.12         119.           9.62E-01  บ   
     บ    L3M4        8.335    .08099         183. ๑     12.     23.7  บ   
     บ              415.634    196.71         119.           9.61E-01  บ   
     บ    L3N5        9.960    .15201         344. ๑     14.    178.4  บ   
     บ              495.448    206.96         128.           9.77E-01  บ   
     บ    L3M1        7.388    .03668          83. ๑     10.     17.3  บ   
     บ              369.120    190.48          99.           9.45E-01  บ   
     บ    L3N1        9.612    .00920          21. ๑     11.     47.1  บ   
     บ              478.356    204.81         126.           9.74E-01  บ   
     บ    L3O1       10.129    .00162           4. ๑     11.    151.2  บ   
     บ              503.749    208.00         129.           9.78E-01  บ   
     บ    L3O5       10.200    .00558          13. ๑     11.     72.5  บ   
     บ              507.236    208.44         129.           9.78E-01  บ   
     บ                                                                 บ   
     บ  4 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
